### ISSUE TITLE
'type_traits' file not found

### ISSUE THREAD

**Issue by skylook** [g-truc/glm#351]
Cannot be compiled with libstdc++ under Xcode 6.3. It shows with error:

thirdparty/glm/detail/type_int.hpp:37:11: 'type_traits' file not found

**Comment by Groovounet** [comment]
I think the issue comes from this code:

```#if GLM_PLATFORM == GLM_PLATFORM_ANDROID
#	define GLM_HAS_CXX11_STL 0
#elif GLM_COMPILER & (GLM_COMPILER_LLVM | GLM_COMPILER_APPLE_CLANG)
#	define GLM_HAS_CXX11_STL __has_include("__config")
#else
#	define GLM_HAS_CXX11_STL ((GLM_LANG & GLM_LANG_CXX0X_FLAG) && \
		((GLM_COMPILER & GLM_COMPILER_GCC) && (GLM_COMPILER >= GLM_COMPILER_GCC48)) || \
		((GLM_COMPILER & GLM_COMPILER_VC) && (GLM_COMPILER >= GLM_COMPILER_VC2013)) || \
		((GLM_COMPILER & GLM_COMPILER_INTEL) && (GLM_COMPILER >= GLM_COMPILER_INTEL15)))
#endif```

Do you know whether it's possible to detection which STL implementation is used? libstdc++ or libc++?

Thanks,
Christophe

**Comment by Groovounet** [comment]
This issue should be fixed in master branch for GLM 0.9.7

Thanks for contributing!
Christophe
